modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -371,20 +393,24 @@ def decision_date_not_in_past(issue, issue_index)
       add_decision_date_error "isn't in the past: #{issue.decision_date_string.inspect}", issue_index
     end
 
+    # rubocop:disable Layout/LineLength
     def add_decision_date_error(string, issue_index)
-      add_error "included[#{issue_index}].attributes.decisionDate #{string}"
-    end
-
-    def add_error(message)
-      errors.add(:base, message)
+      add_error("included[#{issue_index}].attributes.decisionDate #{string}", title: 'Invalid field value',
+                                                                              missing_fields: ['decision date'],
+                                                                              pointer: '/data/attributes/included/items/attributes/properties',
```

Comment:
Same as earlier. Detail should be the "date not in past" message and the pointer should be to the specific index & path that had the bad date.